src/JsonApiDotNetCore.MongoDb/Extensions/MongoIQueryableExtensions.cs

```diff
@@ -1,37 +1,20 @@
-﻿namespace JsonApiDotNetCore.MongoDb.Extensions
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Threading.Tasks;
-    using MongoDB.Driver;
-    using MongoDB.Driver.Linq;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
 
+namespace JsonApiDotNetCore.MongoDb.Extensions
+{
     public static class MongoIQueryableExtensions
     {
-        public static async Task<IReadOnlyList<T>> ToListAsync<T>(this IQueryable<T> queryable)
-        {
-            return await IAsyncCursorSourceExtensions.ToListAsync(ToMongoQueryable(queryable));
-        }
-
-        public static async Task<T> SingleOrDefaultAsync<T>(this IQueryable<T> queryable)
-        {
-            return await IAsyncCursorSourceExtensions.SingleOrDefaultAsync(ToMongoQueryable(queryable));
-        }
+        public static async Task<IReadOnlyList<T>> ToListAsync<T>(this IQueryable<T> queryable) =>
```

Comment:
There is only a single usage of this method, so I think would be better to move it into the repository itself as a private method. That makes the code simpler, because the check for `IMongoQueryable` can be removed. And it no longer hides the fact that `CancellationToken` is not being passed, which it should take.